tracer-core/src/main/java/org/zalando/tracer/DockerGenerator.java

```diff
@@ -0,0 +1,421 @@
+        // Sir Mokshagundam Visvesvaraya - is a notable Indian engineer.  He is a recipient of the Indian Republic's highest honour, the Bharat Ratna, in 1955. On his birthday, 15 September is celebrated as Engineer's Day in India in his memory - https://en.wikipedia.org/wiki/Visvesvaraya
+        "visvesvaraya",
+        // Christiane Nüsslein-Volhard - German biologist, won Nobel Prize in Physiology or Medicine in 1995 for research on the genetic control of embryonic development. https://en.wikipedia.org/wiki/Christiane_N%C3%BCsslein-Volhard
+        "volhard",
+        // Marlyn Wescoff - one of the original programmers of the ENIAC. https://en.wikipedia.org/wiki/ENIAC - https://en.wikipedia.org/wiki/Marlyn_Meltzer
+        "wescoff",
+        // Roberta Williams, did pioneering work in graphical adventure games for personal computers, particularly the King's Quest series. https://en.wikipedia.org/wiki/Roberta_Williams
+        "williams",
+        // Sophie Wilson designed the first Acorn Micro-Computer and the instruction set for ARM processors. https://en.wikipedia.org/wiki/Sophie_Wilson
+        "wilson",
+        // Jeannette Wing - co-developed the Liskov substitution principle. - https://en.wikipedia.org/wiki/Jeannette_Wing
+        "wing",
+        // Steve Wozniak invented the Apple I and Apple II. https://en.wikipedia.org/wiki/Steve_Wozniak
+        "wozniak",
+        // The Wright brothers, Orville and Wilbur - credited with inventing and building the world's first successful airplane and making the first controlled, powered and sustained heavier-than-air human flight - https://en.wikipedia.org/wiki/Wright_brothers
+        "wright",
+        // Rosalyn Sussman Yalow - Rosalyn Sussman Yalow was an American medical physicist, and a co-winner of the 1977 Nobel Prize in Physiology or Medicine for development of the radioimmunoassay technique. https://en.wikipedia.org/wiki/Rosalyn_Sussman_Yalow
+        "yalow",
+        // Ada Yonath - an Israeli crystallographer, the first woman from the Middle East to win a Nobel prize in the sciences. https://en.wikipedia.org/wiki/Ada_Yonath
+        "yonath"
+    };
+
+    /**
+     * @return a random name from the list of adjectives and surnames in this package
+     * formatted as "adjective_surname". For example 'focused_turing'.
+     */
+    @Override
+    public String generate() {
+        do {
+            final UUID uuid = UUID.randomUUID();
+            final int left = Math.abs((int) (uuid.getMostSignificantBits() >> 32));
+            final int right = Math.abs((int) (uuid.getLeastSignificantBits() >> 32));
+            final String name = LEFT[left % LEFT.length] + "_" + RIGHT[right % RIGHT.length];
+
+            // Steve Wozniak is not boring
+            if (!name.equals("boring_wozniak")) {
+                return name;
+            }
+        } while (true);
+    }
+
+    @VisibleForTesting
+    static int maxCombinations() {
+        return LEFT.length * RIGHT.length - 1;
```

Comment:
What's the current number of combinations? I'm not sure this generator is that useful as a normal application would need to create much more traces than you would run docker containers.

I think docker also makes sure that your names are unique by retrying and appending random numbers.